pkg/generators/models/models.go

```diff
@@ -32,12 +31,9 @@ var (
 	validatedTypesRegExp = regexp.MustCompile(`^(\*?[A-Z])|(\[])|(map\[)`)
 )
 
-// something is just something that does not have to be allocated
-type something struct{}
-
 // passedSchemas is a map of pointers to a schema.
 // Used for marking already visited types when recursively resolve `allOf` definitions and merging types.
-type passedSchemas map[*openapi3.SchemaRef]something
+type passedSchemas map[*openapi3.SchemaRef]bool
```

Comment:
why move to a boolean? the empty "something" was intended as a meaningless, but non-empty value, vs the `bool` which has implied meaning.